app/views/content/blog/my-career-change-to-teaching.md

```diff
@@ -0,0 +1,37 @@
+---
+title: My career change to teaching
+author: Ian Wallace
+date: "2022-02-03"
+images:
+  ian:
+    path: "media/images/content/blog/ian-wallace.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/ian-wallace.jpg"
+description: |-
+  Ian Wallace had always wanted to be a teacher but didn’t feel ready when he finished university. After several different jobs and careers he finally took the plunge and hasn’t looked back.
```

Comment:
Meta description is too long - needs to be less than 160 characters. Also good for it to include a call to action, something like 'Read about Ian Wallace's journey into teaching...'

Reply:
Hi @EllieNodder - is this better?

Ian Wallace had always wanted to be a teacher but chose a different path after university. Read about his decision to change career.

Reply:
@christinehitch yep, that's great :)